en/development/testing.rst

```diff
@@ -13,7 +13,7 @@ Installing PHPUnit
 CakePHP uses PHPUnit as its underlying test framework. PHPUnit is the de-facto
 standard for unit testing in PHP. It offers a deep and powerful set of features
 for making sure your code does what you think it does. PHPUnit can be installed
-through using either a `PHAR package <http://phpunit.de/#download>`_ or `Composer
+through using either a `PHAR package <http://phpunit.de/#download>`__ or `Composer
```

Comment:
To avoid some `WARNING: Duplicate explicit target name`.
Sphinx calls it [hyperlink-references](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#hyperlink-references).

The warning appears because we have multiple links (ie : http://phpunit.de/manual/current/en/installation.html#installation.phar.windows, http://phpunit.de/#download) with the same domain name and with different text labels. I found this [here](http://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text)